src/main/java/graphql/schema/idl/SchemaGenerator.java

```diff
@@ -257,13 +259,66 @@ private GraphQLObjectType buildObjectType(BuildContext buildCtx, ObjectTypeDefin
         builder.name(typeDefinition.getName());
         builder.description(buildDescription(typeDefinition));
 
-        typeDefinition.getFieldDefinitions().forEach(fieldDef ->
-                builder.field(buildField(buildCtx, typeDefinition, fieldDef)));
+        //
```

Comment:
This whole Method `buildObjectType` is too long imho. I suggest splitting it up into smaller parts.